apps/services/bff/src/app/modules/ids/ids.service.ts

```diff
@@ -35,60 +33,28 @@ export class IdsService {
   private async postRequest<T>(
     endpoint: string,
     body: Record<string, string>,
-  ): Promise<ApiResponse<T>> {
-    try {
-      const response = await this.enhancedFetch(
-        `${this.issuerUrl}${endpoint}`,
-        {
-          method: 'POST',
-          headers: {
-            'Content-Type': 'application/x-www-form-urlencoded',
-            Authorization: this.createPARAuthorizationHeader(),
-          },
-          body: new URLSearchParams(body).toString(),
-        },
-      )
-
-      const contentType = response.headers.get('content-type') || ''
-
-      if (contentType.includes('application/json')) {
-        const data = await response.json()
-
-        if (!response.ok) {
-          // If error response from Ids is not in the expected format, throw the data as is
```

Comment:
This code was never reached and was a leftover when I switched from fetch to enhancedFetch